app/models/user.rb

```diff
@@ -11,4 +11,25 @@ def self.authenticate(email, password)
     user = User.find_for_authentication(email: email)
     user&.valid_password?(password) ? user : nil
   end
+
+  def get_access_token(client_app_id)
```

Comment:
The method name can be `create_access_token`, as we are creating new everytime we call this.

Reply:
Make perfect sense! 📝
Fixed ;-)